modules/odc-creating-apiserversource.adoc

```diff
@@ -9,16 +9,12 @@
 
 . Navigate to the **Add** page and select **Event Source**.
 . In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-+
-image::apiserversource-odc.png[Create an ApiServerSource]
 . Configure the **ApiServerSource** settings:
 .. Enter `v1` as the **APIVERSION**, and `Event` as the **KIND**.
 // .. Select **Resource** as the **Mode**. **Mode** is the mode that the receive adapter controller runs in. `Ref` sends only the reference to the resource. `Resource` sends the full resource.
 // TODO: clarify what this is used for. Out of scope for this PR since not required.
 .. Select the **Service Account Name** for the service account that you created.
-+
-image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
-.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
+.. Select the **Sink** for the event source. A **Sink** can be either a **Resource** (Broker, Channel or Knative Service) or a **URI**.
```

Comment:
```suggestion
.. Select the **Sink** for the event source. A **Sink** can be either a **Resource**, such as a channel, broker, or service, or a **URI**.
```

Reply:
I'd keep these lowercase unless they're referring to a specific object or text in ODC.

Reply:
Also, no ( ) is part of the OCP docs guidelines afaik :wink:

Reply:
:+1:. btw I love this feature.